tests/foreman/cli/test_host.py

```diff
@@ -819,6 +819,41 @@ def test_positive_list_by_last_checkin(
     assert rhel7_contenthost.hostname in [host['name'] for host in hosts]
 
 
+@pytest.mark.host_create
```

Comment:
I added this because all tests in the module have it, I think it was created by the component owner for collection purposes here https://github.com/SatelliteQE/robottelo/pull/8184 I don't have strong feelings about it

Reply:
I dont see it used anywhere than just applied on few test cases nor CI is filtering test cases based on host_CRUDs. I wont block this PR on it but can we remove these markers now if its not in use @JacobCallahan @tstrych ?